flickit-assessment-core/src/main/java/org/flickit/assessment/core/application/port/out/assessmentreport/CreateAssessmentReportPort.java

```diff
@@ -14,9 +14,21 @@ public interface CreateAssessmentReportPort {
      */
     void persist(Param param);
 
+    /**
+     * Persists a new quick assessment report in an UNPUBLISHED and RESTRICTED state.
```

Comment:
```suggestion
     * Persists a new quick assessment report in an PUBLISHED and RESTRICTED state.
```

البته در راستای کامنت دیگه‌ای که روی این فایل زدم، این متد کلا حذف می‌شه.